components/donation/Donation.component.tsx

```diff
@@ -0,0 +1,171 @@
+          >
+            <Box component="span" m={1}>
+              <Chip
+                label={nodeInfo!.alias}
+                style={{
+                  background: "#" + nodeInfo!.color,
+                }}
+              />
+            </Box>
+          </Grid>
+          <Grid
+            container
+            direction="row"
+            justifyContent="center"
+            alignItems="center"
+          >
+            <QRCode value={`${bolt12}`} size={300} level="H" />
+          </Grid>
+          <Grid
+            container
+            direction="row"
+            justifyContent="center"
+            alignItems="center"
+          >
+            {/* <FormControl
+              variant="outlined"
+              style={{
+                margin: theme.spacing(2),
+                minWidth: 230,
+                textAlign: "center",
+              }}
+            >
+              <InputLabel id="address-select-outlined-label">
+                Address
+              </InputLabel>
+              <Select
+                labelId="address-select-outlined-label"
+                id="address-outlined-select"
+                value={selectAddr.type}
+                onChange={(event: any) =>
+                  setSelectAddr(mapAddress.get(event.target.value))
+                }
+                label="Address"
+                input={<BootstrapInput />}
+              >
+                {nodeInfo!.address.map((address, index) => {
+                  return (
+                    <MenuItem key={index} value={address.type}>
+                      {address.type}
+                    </MenuItem>
+                  );
+                })}
+              </Select>
+            </FormControl> */}
+            <Grid>
+              <SyntaxHighlighter
+                language="json"
+                useInlineStyles={false}
+                customStyle={{
+                  backgroundColor: theme.palette.background.paper,
+                  color: theme.palette.text.primary,
+                  width: 600,
+                  top: "auto",
+                  bottom: 0,
+                  overflow: "scroll",
+                }}
+                codeTagProps={{
+                  style: {
+                    color: theme.palette.text.primary,
+                  },
+                }}
+              >
+                fetchinvoice offer [msatoshi] [quantity] [recurrence_counter]
+                [recurrence_start] [recurrence_label] [timeout] [payer_note]{" "}
+              </SyntaxHighlighter>
+            </Grid>
+            <IconButton
+              onClick={() => navigator.clipboard.writeText(`${bolt12}`)}
+              size="large"
+            >
+              <FileCopyTwoTone />
+            </IconButton>
+          </Grid>
+        </CardContent>
+      </Card>
+      <Grid
+        container
+        direction="row"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <NodeTable show={show} />
```

Comment:
```suggestion
        <OfferTable show={show} />
```

This table should be a `OfferTable` and display all the offers that we have, you can have the list of offers from this command here https://lightning.readthedocs.io/lightning-listoffers.7.html

Please note that we want to display only the active offers (`"active": true,`)